app/src/main/java/com/nervousfish/nervousfish/exceptions/NoBluetoothException.java

```diff
@@ -0,0 +1,24 @@
+package com.nervousfish.nervousfish.exceptions;
+
+/**
+ * Simple {@link RuntimeException} that is used to notify usage of Bluetooth is not avaiable.
+ */
+public class NoBluetoothException extends RuntimeException {
```

Comment:
> instance field access / methods not qualified with this because you like to use this apparently.

Kind of a weird place to give this advice, but thanks anyway 😄 


> Everything from concurrency because it really are good practices...

Again, kinda of a weird place to mention this. Not getting any errors in this class from that...


> Everything from serialization, because serialversionuid...

I [added](https://github.com/ericcornelissen/NervousFish/pull/181/files#diff-a8926c7e0f9739e6fb5fed5d79dedefdR8) a `serialVersionUID` but am not convinced of adding a default `readObject` & `writeObject` method...